### PR TITLE
Update Header Navigation

### PR DESCRIPTION
**Pull request by FrankApiyo**
Fixes issue: #432

### CODE REVIEW

**Comment by FrankApiyo** [comment]
The snapshot sizes have reduced because they now concentrate on ensuring that specific elements of the header have been rendered correctly instead of the whole header

File: src/configs/env.ts [hunk 1]
```diff
@@ -16,6 +16,10 @@ export type ENABLE_IRS = typeof ENABLE_IRS;
 export const ENABLE_FI = process.env.REACT_APP_ENABLE_FI === 'true';
 export type ENABLE_FI = typeof ENABLE_FI;
 
+/** Do you want to enable the HISTORIC_FI? */
+export const ENABLE_HISTORIC_FI = process.env.REACT_APP_ENABLE_HISTORIC_FI === 'false';
+export type ENABLE_HISTORIC_FI = typeof ENABLE_HISTORIC_FI;
```
Comment (moshthepitt) [inline src/configs/env.ts]: Please add this to sample.env

**Comment by moshthepitt** [comment]
Please also fix the conflict

**Comment by moshthepitt** [comment]
Seems that this is both superseded and replaced by https://github.com/onaio/reveal-frontend/pull/546

**Pull request closed without merging**